app/src/main.js

```diff
@@ -53,5 +53,18 @@ init(urlParams).then(({ store, router, apolloProvider }) => {
         StatusBar.show()
       }
     },
+    watch: {
+    '$route': {
+      handler: function () {
+          if (this.$route.name === `notifications`) {
```

Comment:
let's change this soon. ok for now

Reply:
i don't think we need this actually - i will remove this watch function